### PR TITLE
feat(WebexMeeting): theme the Webex meeting component

### PR DESCRIPTION
**Pull request by karinasigartau0798**
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-254217

### CODE REVIEW

**Comment by jonniespratley** [comment]
Do we need defaults for css vars?

**Comment by karinasigartau0798** [comment]
> Do we need defaults for css vars?

No, it is not necessary to have defaults for these types of variables.

File: src/themes/dark.scss [hunk 1]
```diff
@@ -1,6 +1,8 @@
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-theme-dark {
   --wxc-meeting-background: linear-gradient(180deg, #1A1A1A 0%, #0F0F0F 100%);
   --wxc-text-color: rgba(255, 255, 255, 0.95);
+  --wxc-meeting-border: 1px solid rgba(255, 255, 255, 0.3);
```
Comment (cipak) [inline src/themes/dark.scss:4]: please put the --wxc-text-color var at the top and group the 3 meeting variables after a blank line:
```css
--wxc-text-color:

--wxc-meeting-background
--wxc-meeting-border
--wxc-meeting-box-shadow

Comment (cipak) [reply src/themes/dark.scss:4]: also, use scss colors

**Review by cipak — APPROVED** [review]

**Pull request merged**